api-reference/10 UI Components/dxVectorMap/1 Configuration/layers/paletteIndex.md

```diff
@@ -4,7 +4,12 @@ type: Number
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The position of a color in the **[palette[]** array. Should not exceed the value of the [paletteSize](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/layers/#paletteSize) property. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Use this property to specify the color for each [layer element](/Documentation/ApiReference/UI_Components/dxVectorMap/Map_Elements/Layer_Element/) within the [customize](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/layers/#customize) function.
```

Comment:
When you mention the customize function, it doesn't tell anything, because the reader doesn't see the code.

```suggestion
Use this property to specify a different color for each [layer element](/Documentation/ApiReference/UI_Components/dxVectorMap/Map_Elements/Layer_Element/). Refer to the [palette](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/layers/#palette) description for a code example.
```